.github/workflows/release.yml

```diff
@@ -0,0 +1,152 @@
+        run: |
+          AC_API_KEY_SECRET_PATH="$RUNNER_TEMP/key.p8"
+          echo "APP_STORE_CONNECT_API_KEY_KEY_FILEPATH=$AC_API_KEY_SECRET_PATH" >> $GITHUB_ENV
+          echo -n "$AC_API_KEY_SECRET_BASE64" | base64 --decode --output "$AC_API_KEY_SECRET_PATH"
+
+      - name: Install the google play key material
+        env:
+          GOOGLE_PLAY_API_JWT_BASE64: ${{ secrets.GOOGLE_PLAY_API_JWT_BASE64 }}
+          GOOGLE_PLAY_KEYSTORE_BASE64: ${{ secrets.GOOGLE_PLAY_KEYSTORE_BASE64 }}
+        run: |
+          GOOGLE_PLAY_API_JWT_PATH="$RUNNER_TEMP/gp_api.json"
+          echo "GOOGLE_PLAY_API_JWT_PATH=$GOOGLE_PLAY_API_JWT_PATH" >> $GITHUB_ENV
+          echo -n "$GOOGLE_PLAY_API_JWT_BASE64" | base64 --decode --output "$GOOGLE_PLAY_API_JWT_PATH"
+          
+          GOOGLE_PLAY_KEYSTORE_PATH="$RUNNER_TEMP/gp_signing.jks"
+          echo "GOOGLE_PLAY_KEYSTORE_PATH=$GOOGLE_PLAY_KEYSTORE_PATH" >> $GITHUB_ENV
+          echo -n "$GOOGLE_PLAY_KEYSTORE_BASE64" | base64 --decode --output "$GOOGLE_PLAY_KEYSTORE_PATH"
+
+      - name: Get build name and number, install dependencies
+        run: |
+          go install golang.org/x/mobile/cmd/gomobile@latest
+          gomobile init
+          
+          flutter pub get
+          
+          touch env.sh
+          
+          cd android
+          fastlane release_build_number
+          echo "BUILD_NUMBER=$(cat ../release_build_number)" >> $GITHUB_ENV
+          echo "BUILD_NAME=${GITHUB_REF#refs/tags/v}" >> $GITHUB_ENV
+
+      - name: Build iOS
+        env:
+          MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
+        run: |
+          cd ios
+          pod install
+          fastlane build
+
+      - name: Collect iOS artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: MobileNebula.ipa
+          path: ios/MobileNebula.ipa
+          retention-days: 5
+
+      - name: Build Android
+        env:
+          ANDROID_NDK_HOME: ${{ steps.setup-ndk.outputs.ndk-path }}
+          GOOGLE_PLAY_KEYSTORE_PASSWORD: ${{ secrets.GOOGLE_PLAY_KEYSTORE_PASSWORD }}
+        run: |
+          flutter build appbundle --build-number="$BUILD_NUMBER" --build-name="$BUILD_NAME"
+
+      - name: Collect Android artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: MobileNebula.aab
+          path: build/app/outputs/bundle/release/app-release.aab
+          retention-days: 5
+
+      - name: Publish to iOS TestFlight
+        env:
+          APP_STORE_CONNECT_API_KEY_KEY_ID: ${{ secrets.AC_API_KEY_ID }}
+          APP_STORE_CONNECT_API_KEY_ISSUER_ID: ${{ secrets.AC_API_KEY_ISSUER_ID }}
+        run: |
+          cd ios
+          fastlane release
+
+      - name: Publish to Android internal track
+        run: |
+          cd android
+          fastlane release
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't see this secret in the repo

Reply:
This one is provided by github on every run